core/resources/l10n/messages.properties

```diff
@@ -1859,8 +1859,8 @@ Warning.RECOVERY_HIGH_SPEED = Recovery device deployment at high speed
 Warning.NO_RECOVERY_DEVICE = No recovery device defined in the simulation.
 Warning.TUMBLE_UNDER_THRUST = Stage began to tumble under thrust.
 Warning.EVENT_AFTER_LANDING = Flight Event occurred after landing:  
-Warning.ZERO_LENGTH_BODY = Zero length bodies may not result in accurate simulations.
-Warning.ZERO_RADIUS_BODY = Zero length bodies may not result in accurate simulations.
+Warning.ZERO_LENGTH_BODY = Zero-length/diameter bodies may not simulate accurately.
```

Comment:
I would remove the redundancy and keep only one value, e.g. rename it to e.g. `Warning.ZERO_VOLUME_BODY` and then change `FinSetCalc.java` line 104 to:

```Java
if ((bodyLength < EPSILON) || (bodyRadius < EPSILON)) {
	// Add warnings: Phantom Body
	warnings.add(Warning.ZERO_VOLUME_BODY);
} else if( (0 < bodyRadius) && (thickness > bodyRadius / 2)){
```

Reply:
Warning.java also required a change, along with the translations (that will need to be verified/modified).